content-src/components/PocketStories/PocketStories.js

```diff
@@ -71,7 +72,12 @@ const PocketStories = React.createClass({
     return (<section className="pocket-stories spotlight">
       <h3 className="section-title">
         <FormattedMessage id="header_stories" />
-        <span className="pocket-logo" />
+        <span className="section-title-logo" >
+          <span className="pocket-logo-text">
+              <FormattedMessage id="header_stories_by" />
+          </span>
+          <span className="pocket-logo" />
```

Comment:
Can you include some text in here for screen readers (just add `<span className="sr-only">Pocket</span>` as a child of these element)